supply/lib/supply/uploader.rb

```diff
@@ -34,9 +34,9 @@ def perform_upload
 
     def promote_track
       version_codes = client.track_version_codes(Supply.config[:track])
-      client.update_track(Supply.config[:track], 1.0, nil)
+      client.update_track(Supply.config[:track], 0.1, nil)
```

Comment:
I wasn't sure about that. the value here does not really matter as long as it is between 0,05 and 0.5. But if we put there the value the user provides, it looks like the value matters. So I simply put some fixed value there.
But with a fixed value it is some kind of "magic" number...
So if you want me to change it, no problem. I'm fine with both ;)
I could also add a comment explaining that it does not matter which value is passed there